addons/resource.language.en_gb/resources/strings.po

```diff
@@ -20024,7 +20024,16 @@ msgctxt "#36421"
 msgid "Bypassing VDPAU mixer saves resources on low power systems but slightly reduces picture quality."
 msgstr ""
 
-#empty strings from id 36422 to 36423
+#: system/settings/settings.xml
+msgctxt "#36422"
+msgid "Update trusted add-ons from"
+msgstr ""
+
+#. Description of setting with label #36422 "Update trusted add-ons from"
+#: system/settings/settings.xml
+msgctxt "#36423"
+msgid "By default, add-ons from trusted repositories will be prevented from being updated from private repositories. For cases such as updating from an add-ons beta repository this option can be switched to [Any repositories] (bear in mind this is a less secure option)."
```

Comment:
Open question: Is this only or even mainly about security? Isn't the problem also that it will cause incompatibility and crashes?

Reply:
Yes, I guess we could extend the text. The setting was added specifically to allow beta versions of add-ons. Maybe change the part in brackets to:

`(bear in mind this is a less secure option and enabling it could cause incompatibility and crashes)`???

Reply:
this is only the helptext, if no one objects i‘ll add it this evening (ofc. we‘re doing our best that this will cause neither incompat. nor crashes, but you never know)